src/microsoft-dotnet-interactive/src/htmlKernel.ts

```diff
@@ -45,7 +46,8 @@ export interface HtmlDomFragmentInserterConfiguration {
     getOrCreateContainer?: () => HTMLElement,
     updateContainerContent?: (container: HTMLElement, htmlFragment: string) => void,
     createMutationObserver?: (callback: MutationCallback) => MutationObserver,
-    normalizeHtmlFragment?: (htmlFragment: string) => string
+    normalizeHtmlFragment?: (htmlFragment: string) => string,
+    jsEvaluator?: (js: string) => void
```

Comment:
is this ever explicitly set anywhere?

Reply:
17 lines later ...

Reply:
I meant by specifying it in the config value.  I don't ever see it being specified, so why not leave it as `eval(...)`?

Reply:
if done that way is the eval of node during testing and not the one from the browser engine